aea/cli_gui/__init__.py

```diff
@@ -157,19 +151,12 @@ def remove_local_item(agent_id, item_type, item_id):
 
 def get_local_items(agent_id, item_type):
```

Comment:
@dishmop here and in other places you're missing the typing annotations. Please add them for consistency and readability. Note, in the `cli` we cannot use them as this messes with the `click` module. But here I suspect it is ok, unless you think otherwise?

Reply:
OK I'll put those in

Reply:
hmm actually I've now put then in the arguments list - but not on the return types as the functions often do return different structures depending on whether they pass or fail.